metals/src/main/scala/scala/meta/internal/metals/debug/DebugProxy.scala

```diff
@@ -45,7 +45,8 @@ private[debug] final class DebugProxy(
       exitStatus.trySuccess(Restarted)
       outputTerminated = true
       server.consume(message)
-
+    case null =>
```

Comment:
Can we pack it into an Option like:
```scala
{ message =>
    Option(message) match {
      case _ if cancelled.get() =>
      // ignore
      case Some(OutputNotification()) if outputTerminated =>
      // ignore. When restarting, the output keeps getting printed for a short while after the
      // output window gets refreshed resulting in stale messages being printed on top, before
      // any actual logs from the restarted process

      case Some(msg) =>
        client.consume(msg)
      case None =>
    }
  }
```

Reply:
This change is unrelated to the PR, let's refactor it separately. I just wanted to fix a NPE